src/App.tsx

```diff
@@ -67,18 +67,14 @@ export const App: React.FC = () => {
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length === 0 && (
+                {author && isLoaded && !hasError && posts.length === 0 && (
```

Comment:
we can get empty list message with no loader status and errors.
If the loader is active or there are errors, the list will be empty
```suggestion
                {(author && !posts.length) && (
```